cluster/container.go

```diff
@@ -10,3 +16,50 @@ type Container struct {
 	Info   dockerclient.ContainerInfo
 	Engine *Engine
 }
+
+// Status returns a human-readable description of the state
+// Stoken from docker/docker/daemon/state.go
+func (c *Container) Status() string {
+	s := c.Info.State
+	if s.Running {
+		if s.Paused {
```

Comment:
I just took this from `docker/docker/daemon/state.go` without touching :)

The second statement uses `FinishedAt` though, so we could only re-use it twice, not sure if it's worth it.